src/dialogs/transcodedialog.ui

```diff
@@ -140,6 +137,61 @@
      </property>
     </widget>
    </item>
+   <item>
+    <widget class="QWidget" name="advancedWidget" native="true">
+     <layout class="QVBoxLayout" name="verticalLayout_2">
+      <item>
+       <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
+        <item row="1" column="1">
+         <widget class="QCheckBox" name="fpsCheckBox">
+          <property name="toolTip">
+           <string>Enable frame rate override.</string>
+          </property>
+          <property name="text">
+           <string>Override Frame Rate</string>
+          </property>
+         </widget>
+        </item>
+        <item row="0" column="0" colspan="2">
+         <widget class="Line" name="line">
+          <property name="orientation">
+           <enum>Qt::Horizontal</enum>
+          </property>
+         </widget>
+        </item>
+        <item row="2" column="1">
+         <widget class="FrameRateWidget" name="fpsWidget" native="true">
+          <property name="toolTip">
+           <string>Override the frame rate to a specific value.</string>
+          </property>
+         </widget>
+        </item>
+        <item row="2" column="0">
+         <widget class="QLabel" name="fpsLabel">
+          <property name="text">
+           <string>Frames/sec</string>
+          </property>
+         </widget>
+        </item>
+        <item row="3" column="0">
+         <widget class="QLabel" name="frcLabel">
+          <property name="text">
+           <string>Frame Rate Conversion</string>

```

Comment:
Please change the casing on this label to uppercase first word only. See https://shotcut.org/notes/ui-conventions/

Reply:
fixed